code/game/objects/machinery/floodlight.dm

```diff
@@ -80,226 +80,131 @@
 	set_light(brightness_on)
 
 /obj/machinery/floodlightcombat
-	name = "Armoured floodlight"
+	name = "armoured floodlight"
```

Comment:
to be 'armored' or to be armoured'
color or colour
grey or gray

Reply:
What is the issue? Both are acceptable

Reply:
none, just musing cuz its funny.